src/package-tree.ts

```diff
@@ -1,7 +1,14 @@
-interface PointOfInterest {
+export interface PointOfInterest {
   type: string;
```

Comment:
You can punt this to another PR if you like, but "type" but be better expressed as an enum rather than a string.

https://www.typescriptlang.org/docs/handbook/enums.html